react/utils/sla.ts

```diff
@@ -0,0 +1,7 @@
+import { DeliveryOption, PickupOption } from 'vtex.checkout-graphql'
+
+export const getName = (shippingOption: DeliveryOption | PickupOption) => {
+  return shippingOption.deliveryChannel === 'pickup-in-point'
+    ? (shippingOption as PickupOption).friendlyName
```

Comment:
you could use type guards to avoid this "cast", which I think may be more elegant

```ts
const isPickupOption = (
  shippingOption: DeliveryOption | PickupOption
): shippingOption is PickupOption => {
  return shippingOption.deliveryChannel === 'pickup-in-point'
}
```

then you could just use this function and typescript would automatically cast the item to `PickupOption`

```suggestion
  return isPickupOption(shippingOption)
    ? shippingOption.friendlyName
```

Reply:
Noice! Done.